.clang-tidy

```diff
@@ -116,5 +121,11 @@ CheckOptions:
     value:           'false'
   - key:             google-readability-function-size.StatementThreshold
     value:           '800'
+  - key:             cppcoreguidelines-narrowing-conversions.WarnOnIntegerToFloatingPointNarrowingConversion
+    value:           'true'
+  - key:             readability-function-cognitive-complexity.DescribeBasicIncrements
+    value:           'false'
```

Comment:
Why disable this? Does it improve the report in some way?

Reply:
Given that the check produces warnings that we often ignore, it's better to have this off by default to avoid filling up the report with information we usually don't care about. It can always be overridden via argument.